modules/tinymce/src/themes/silver/main/ts/modes/ScrollingContext.ts

```diff
@@ -52,19 +52,24 @@ export const detect = (popupSinkElem: SugarElement<HTMLElement>): Optional<Scrol
 export const detectWhenSplitUiMode = (editor: Editor, popupSinkElem: SugarElement<HTMLElement>): Optional<ScrollingContext> =>
   Options.isSplitUiMode(editor) ? detect(popupSinkElem) : Optional.none();
 
+const isFullscreen = (editor: Editor): boolean =>
```

Comment:
I wonder if we could make this detection more broad it more that if the `editor.getContainer()` is inside a `position: fixed` element then we should use the window bounds? I think cross plugin API calls should be kept at a minimum if possible. I guess one could also try to check for the `tox-fullscreen` class however that is also kind of plugin specific.

Reply:
initially I was thinking to create a new editor's api like `editor.isFullscreen`, but probably trying to put the logic inside `editor.getContainer()` could have more sense, I'll try it

Reply:
umh, maybe is not a good idea change `editor.getContainer()`, also I don't see how to do a different check because the check is done in [popupSinkBounds](https://github.com/tinymce/tinymce/blob/main/modules/tinymce/src/themes/silver/main/ts/Theme.ts#L57) that make sense to have the check for a fixed container with `ScrollingContext.detectWhenSplitUiMode`

Reply:
Maybe the check is fine then.